packages/cdk/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "engines": {
+    "node": "^14",
+    "npm": "^6"
+  },
```

Comment:
I don't think we should restrict customers to only v14/v6 of Node/NPM. The lockfile version we use is forwards compatible with all current Node/NPM releases.

If you want to set a minimum version (ie `>=14`), I'm all for it, but this is overkill and restrictive to our users.

Reply:
Agree. Final revision removes the strict engine rule. Only locking the lockFileversion